src/etc/inc/filter.inc

```diff
@@ -4382,27 +4382,34 @@ function filter_generate_ipsec_rules($log = array()) {
 			/* Add rules to allow IKE to pass */
 			$shorttunneldescr = substr($descr, 0, 35);
 			// don't add "pass out" rules where $rgip is any as it will over-match and often break VPN clients behind the system in multi-WAN scenarios. redmine #5819
-			if ($rgip != " any ") {
-				$ipfrules .= "pass out {$log['pass']} $route_to proto udp from (self) to {$rgip} port = 500 tracker {$increment_tracker($tracker)} keep state label \"IPsec: {$shorttunneldescr} - outbound isakmp\"\n";
+			if ($rgip != " any " ) {
+				// don't add "pass out" if NAT-T is forced
+				if ( isset($ph1ent['ikeport']) || !isset($ph1ent['nattport']) ) {
```

Comment:
I will remove this test as well as the one for the esp but this is not exactly the same kind of forcing. When forcing with the port fields, the port 500 won't be used at all by the tunnel, so the rule is not needed. When forcing with the NAT Traversal option, the nat detection is faked but the port 500 is still used to initiate the connection. That's why I didn't add the test on nat_traversal there.

Since I'm here, I have a question about the "resolve conversation" buttons: who should mark each item resolved ? me or the reviewer ?

I also took a good look at the style guide and developer style guide (thx), I'll try to do a better job next time.